elytron/src/main/java/org/wildfly/extension/elytron/ElytronSubsystemTransformers.java

```diff
@@ -136,7 +139,12 @@ public void registerTransformers(SubsystemTransformerRegistration registration)
 
     private static void from15_1(ChainedTransformationDescriptionBuilder chainedBuilder) {
         ResourceTransformationDescriptionBuilder builder = chainedBuilder.createBuilder(ELYTRON_15_1_0, ELYTRON_15_0_0);
-
+        builder.addChildResource(PathElement.pathElement(DISTRIBUTED_REALM))
+                .getAttributeBuilder()
+                .setDiscard(DiscardAttributeChecker.UNDEFINED, IGNORE_UNAVAILABLE_REALMS)
+                .setDiscard(DiscardAttributeChecker.UNDEFINED, EMIT_EVENTS)
+                .addRejectCheck(RejectAttributeChecker.DEFINED, IGNORE_UNAVAILABLE_REALMS)
```

Comment:
This can be changed to:

`.addRejectCheck(new RejectAttributeChecker.SimpleRejectAttributeChecker(ModelNode.TRUE), IGNORE_UNAVAILABLE_REALMS)`

(since `false` was the existing behaviour)